### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by aviars**

### CODE REVIEW

File: apps/accounts/emails.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
+import random
```
Comment (whytheplatypus) [inline apps/accounts/emails.py]: would you update this to just be the `template` changes? It looks like there are ways of doing email verification, mfa, etc. leaning on django. e.g. https://medium.com/@frfahim/django-registration-with-confirmation-email-bb5da011e4ef

File: vmi/settings.py [hunk 2]
```diff
@@ -134,3 +149,67 @@
...
+    'DJANGO_ORGANIZATION_TITLE',
+    'Alliance for Better Health')
+ORGANIZATION_URI = env('DJANGO_ORGANIZATION_URI', 'https://abhealth.us')
+POLICY_URI = env(
+    'DJANGO_POLICY_URI',
+    'https://abhealth.us')
+POLICY_TITLE = env('DJANGO_POLICY_TITLE', 'Privacy Policy')
+TOS_URI = env('DJANGO_TOS_URI', 'https://abhealth.us')
+TOS_TITLE = env('DJANGO_TOS_TITLE', 'Terms of Service')
+TAG_LINE_1 = env('DJANGO_TAG_LINE_1', 'Share your health data')
```
Comment (whytheplatypus) [inline vmi/settings.py]: These `TAG_LINE`s should just be one (or an array if needed)

File: vmi/settings.py [hunk 3]
```diff
@@ -134,3 +149,67 @@
...
+TAG_LINE_2 = env('DJANGO_TAG_LINE_2',
+                 'with applications, organizations, and people you trust.')
+EXPLAINATION_LINE = 'This service allows Medicare beneficiaries to connect their health data to applications of their choosing.'
+EXPLAINATION_LINE = env('DJANGO_EXPLAINATION_LINE ', EXPLAINATION_LINE)
```
Comment (whytheplatypus) [inline vmi/settings.py]: dup, can be joined with the one above rather than referencing it.

File: vmi/settings.py [hunk 4]
```diff
@@ -134,3 +149,67 @@
...
+DEVELOPER_DOCS_TITLE = "Developer Documentation"
+DEVELOPER_DOCS = "Developer Docs"
+DEFAULT_DISCLOSURE_TEXT = """
+    Unauthorized or improper use of this
+    system or its data may result in disciplinary action, as well as civil
+    and criminal penalties. This system may be monitored, recorded and
+    subject to audit.
+    """
+
+DISCLOSURE_TEXT = env('DJANGO_PRIVACY_POLICY_URI', DEFAULT_DISCLOSURE_TEXT)
```
Comment (whytheplatypus) [inline vmi/settings.py]: Can be joined with `DEFAULT_DISCLOSURE_TEXT`

File: vmi/settings.py [hunk 5]
```diff
@@ -134,3 +149,67 @@
...
+    'https://abhealth.us')
+POLICY_TITLE = env('DJANGO_POLICY_TITLE', 'Privacy Policy')
+TOS_URI = env('DJANGO_TOS_URI', 'https://abhealth.us')
+TOS_TITLE = env('DJANGO_TOS_TITLE', 'Terms of Service')
+TAG_LINE_1 = env('DJANGO_TAG_LINE_1', 'Share your health data')
+TAG_LINE_2 = env('DJANGO_TAG_LINE_2',
+                 'with applications, organizations, and people you trust.')
+EXPLAINATION_LINE = 'This service allows Medicare beneficiaries to connect their health data to applications of their choosing.'
+EXPLAINATION_LINE = env('DJANGO_EXPLAINATION_LINE ', EXPLAINATION_LINE)
+USER_DOCS_URI ="https://abhealth.us"
```
Comment (whytheplatypus) [inline vmi/settings.py]: We can group these. e.g.
```
USER_DOCS = {
  "URL": <>,
  "TITLE": <>,
  "SHORT_TITLE": <>,
}
```

File: vmi/settings.py [hunk 6]
```diff
@@ -134,3 +149,67 @@
...
+USER_DOCS_TITLE = "User Documentation"
+USER_DOCS = "USer Docs"
+# LINKS TO DOCS
+DEVELOPER_DOCS_URI = "https:/abhealth.us"
```
Comment (whytheplatypus) [inline vmi/settings.py]: same as `USER_DOCS`

**Comment by aviars** [comment]
@whytheplatypus  Is this okay to go ahead and merge?

File: vmi/urls.py [hunk 7]
```diff
@@ -43,10 +44,13 @@
         name="authorized-token-delete"),
 ]
 
+
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('o/', include((oauth2_management_urlpatterns + oauth2_base_urlpatterns, 'oauth2_provider'))),
```
Comment (whytheplatypus) [inline vmi/urls.py]: looks like this change got lost along the way.

File: apps/accounts/urls.py [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+                    forgot_password,activation_verify)
+
+from .sms_mfa_views import mfa_login, mfa_code_confirm
+
+# Copyright Videntity Systems Inc.
+
+urlpatterns = [
+    url(r'^logout', mylogout, name='mylogout'),
+    url(r'^settings', account_settings, name='account_settings'),
+    url(r'^mfa-login', mfa_login, name='mfa_login'),
```
Comment (whytheplatypus) [inline apps/accounts/urls.py]: I think we need a straight `/accounts/login/` path somewhere in order for the flow to work.

**Pull request merged**